dspace-spring-rest/src/main/java/org/dspace/app/rest/repository/EPersonRestRepository.java

```diff
@@ -34,9 +48,52 @@
 public class EPersonRestRepository extends DSpaceRestRepository<EPersonRest, UUID> {
     EPersonService es = EPersonServiceFactory.getInstance().getEPersonService();
 
+    @Autowired
+    AuthorizeService authorizeService;
+
     @Autowired
     EPersonConverter converter;
 
+    @Override
+    protected EPersonRest createAndReturn(Context context)
+            throws AuthorizeException {
+        // this need to be revisited we should receive a mock EPersonRest as input
+        HttpServletRequest req = getRequestService().getCurrentRequest().getHttpServletRequest();
+        ObjectMapper mapper = new ObjectMapper();
+        EPersonRest mock = null;
```

Comment:
The variable name "mock" made me think I was looking at a test class.

Reply:
I'll admit, I thought the first thing initially.  It probably should be named something else, like simply "epersonRest" or similar.  I didn't feel strongly enough to comment on this, but since both @terrywbrady and I felt this is odd, maybe it should be simply changed.